.circleci/config.yml

```diff
@@ -681,13 +682,14 @@ workflows:
 #          requires: ["test-Win.Chrome"]
 #          browsers: "Mac.Firefox"
 #          notify_on_failure: true
-      - test_oldies:
-          name: "test-Win.IE8"
-          requires: ["test-Win.Chrome"]
-          browsers: "Win.IE8"
-          notify_on_failure: true
+      # - test_oldies:
```

Comment:
If we go down this path we should  maybe use [a filter](https://circleci.com/docs/configuration-reference/#jobfilters) here instead. Then it could not potentially stop the tests on master if it got merged there accidentally. 

But I'll also have a quick looksee at the tests.